Add ECFT-VS pipeline project (#137)
volkamerlab/volkamerlab_org: content/projects/ecft-vs-pipeline.md

---

> title: ECFT-VS Pipeline

Reviewer: I would put the hyphen between 'VS' and 'pipeline' rather than between 'ECFT-VS'. Like this, the 'VS' might read like part of the protein name.

date: 2023-04-15
weight: 10
nav: false
# Uncomment sections below to enable them in your project
#external_resources:
#- name: ECFT-VS-pipeline Repository
#  link: https://github.com/volkamerlab/ECFT-VS-pipeline
#  icon: github
people:
- key: hamza.ibrahim
- key: michael.backenkoehler 
- key: andrea.volkamer
cooperation:
- name: Helmholtz-Institute for Pharmaceutical Research Saarland, HIPS
  link: https://www.helmholtz-hips.de/en/
---

> Finding the optimal docking pipeline for a target is challenging. Our project aims to develop a data-driven pipeline to find new hits for Energy coupling factor (ECF) transporters through the optimization of various structure-based virtual screening workflows.

Reviewer 2: ```suggestion
Finding the optimal docking pipeline to identify novel ligands for a target of interest is challenging. Our project aims to develop a data-driven pipeline to find new hits for energy-coupling factor (ECF) transporters through the optimization of various structure-based virtual screening workflows.
```

<!--more-->

> Energy coupling factor (ECF) transporters are novel, promising antimicrobial targets which mediate micronutrient transport into the cell. Therefore this protein is a promising drug target. Absence of ligands with a co-crystallized protein with ligand and its cryptic binding pocket make it a challenging target. However, previous computational work at the Helmholtz-Institute for Pharmaceutical Research Saarland (HIPS) revealed an opening pocket in ECF transporters for pantothenate.

Reviewer 2: ```suggestion
Energy-coupling factor transporters (ECFTs) are novel, promising antimicrobial targets which mediate micronutrient transport into the cell. Absence of co-crystallized protein-ligand complexes and its cryptic binding pocket make ECFT a challenging target. However, previous computational work at the Helmholtz-Institute for Pharmaceutical Research Saarland (HIPS) and collaboration partners revealed an opening pocket in ECFT for pantothenate, which is used as basis for this study.
```

Reviewer 2: @hamzaibrahim21 add a link to the study

Author: @AndreaVolkamer Thanks for the feedback. I've added the link to the study as a reference. Please let me know if I should add it in another way, e.g. add to _publications_ section.

> To virtually screen a large in-house data of HIPS, we develop a structure-based, data-driven pipeline. In this approach we identify an efficient docking pipeline to our target to propose promising novel antibacterial agents, which can be tested further.

Reviewer 2: ```suggestion
To virtually screen a large in house or freely available data sets, we develop a structure-based, data-driven pipeline. In this approach we identify an efficient docking pipeline for the target of interest to propose promising novel antibacterial agents, which can be tested further.
```

> Previous computational work at Helmholtz-Institute for Pharmaceutical Research Saarland (HIPS) revealed an opening pocket in ECF-T for pantothenate. To virtually screen a large in-house data of HIPS, we develop a structure-based, data-driven pipeline. This approach identifies the optimal docking pipeline to our target to find most promising novel antibacterial agents, which will be tested further at HIPS.

Reviewer: ```suggestion
Previous computational work at the Helmholtz-Institute for Pharmaceutical Research Saarland (HIPS) revealed an opening pocket in ECF-T for pantothenate. To virtually screen a large in-house data of HIPS, we develop a structure-based, data-driven pipeline. In this approach we identify an efficient docking pipeline to our target to propose promising novel antibacterial agents, which can be tested further.
```

---

> title: ECFT VS-Pipeline

Reviewer: I just realized that it's ECFT up here and ECF-T in the text below

Author: I've updated the abbreviation in the text. Generally should be ECF transporters. However, to keep the title short, should it be ECFT ?

Reviewer: Yes, we can do it like that.
